training/src/main/java/wolox/training/models/User.java

```diff
@@ -0,0 +1,91 @@
+package wolox.training.models;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import wolox.training.exceptions.BookAlreadyOwnedException;
+
+@Entity (name = "users")
+@ApiModel(description = "Users from LibraryAPI")
+public class User {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private long id;
+
+    @Column(nullable = false)
+    private String username;
+
+    @Column(nullable = false)
+    private String name;
+
+    @Column(nullable = false)
+    private LocalDate birthdate;
```

Comment:
The `@ ApiModelProperty` annotation for the other attributes are missing. Also, add the `required` argument to the annotation

Reply:
Fixed annotations in User.java